Back End/MemorizeWords/MemorizeWords/Api/ApplicationVariables.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace MemorizeWords.Api
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ApplicationVariables : ControllerBase
```

Comment:
convert this api to minimal api and do as same as WordApiInitializer

Reply:
Create a class Called ApplicationVariableApiInitializer and initialize that class like WordApiInitializer